tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+export const DEBUGGER_FEATURE_KEY = 'debugger';
+
+// TODO(cais): Maybe deduplicate this with tensorboard/webapp.
+export enum DataLoadState {
```

Comment:
You should certainly dedup ;) also this code, as discussed in offline, should move to webapp so that would be quite natural.

Reply:
> tensorboard/webapp/types/api.ts

I don't think the LoadState belongs in there. I would love it if it gets moved to tensorboard/webapp/types/data.ts or something to that effect.

Reply:
Done. I created a new file in the same directory (tensorboard/webapp/types/data.ts) and moved the `DataLoadState` and `LoadState` interfaces to that file. Changes are made accordingly in tensorboard/webapp and in tensorboard/plugins/debugger_v2.